examples/Makefile

```diff
@@ -511,6 +511,8 @@ PHYSICS = \
     physics/physics_restitution \
     physics/physics_shatter
 
+EFFECTS = \
```

Comment:
I'd prefer to avoid a new category at this moment, could it be added as `shapes_topdown_lights`?